pkg/storage/s3/s3.go

```diff
@@ -224,6 +240,18 @@ func (d *driver) getCABundle() (string, error) {
 	}
 }
 
+// useDualStack returns true if the driver should use dual-stack endpoints
+func (d *driver) useDualStack() (bool, error) {
+	if d.Config.RegionEndpoint != "" {
+		return true, nil
+	}
+	ok, err := regionHasDualStackS3(d.Config.Region)
+	if err != nil {
+		return true, fmt.Errorf("failed to determine if region %s has dual stack S3: %w", d.Config.Region, err)
```

Comment:
probably a nit: maybe return `false` here with the error? although that value isn't used by the caller on errors, it would be more inline with what `regionHasDualStackS3` returns with an error.

Reply:
I thought it's safer to assume that is supports IPv6. But as it won't be used, it's better to return false there, otherwise it's just confusing.

I'll fix it, I need to fix linter errors too.